### PR TITLE
SYN-6037: `view.exec` more explicitly doc separation of events

### PR DESCRIPTION
**Pull request by MichaelSquires**
docs: Updated command-line help for `view.exec`.
docs: Updated user-guide for `view.exec` command to include more detailed information about how the sub-runtime works.

### CODE REVIEW

**Comment by MichaelSquires** [comment]
Here's what the rendered doc update looks like:
image

**Review by invisig0th — DISMISSED** [review]

File: docs/synapse/userguides/storm_ref_cmd.rstorm [hunk 1]
```diff
@@ -2468,6 +2468,35 @@ view.exec
...
+**Behavior and Limitations**
+
+The ``view.exec`` command creates its own sub-runtime to execute a storm query in a different view.
+This sub-runtime creates a firm separation boundary between the source view and the destination
+view. This means nodes do not pass in or out across a ``view.exec`` boundary. Pipelines, events,
+messages, etc will NOT pass from the view.exec sub-runtime to the parent runtime or vice-versa. This
+includes ``$lib.print(...)``, ``$lib.warn(...)``, and other functions that may print to the CLI.
+
+It is possible for variables from the source runtime to pass into the sub-runtime created by
+``view.exec``. Variables from the source runtime may also be assigned to in the destination view.
```
Comment (therealsilence) [inline docs/synapse/userguides/storm_ref_cmd.rstorm:2480]: Missing something? "...may also be assigned to in the destination..." feels weird.

File: docs/synapse/userguides/storm_ref_cmd.rstorm [hunk 2]
```diff
@@ -2468,6 +2468,35 @@ view.exec
...
+
+  // Move some tagged nodes to another view
+  inet:fqdn#foo.bar $fqdn=$node.value() | view.exec 95d5f31f0fb414d2b00069d3b1ee64c6 { [ inet:fqdn=$fqdn ] }
+
+Here we have ``inet:fqdn`` nodes with the tag ``#foo.bar`` being lifted and their value (not the
+node) is saved into the ``$fqdn`` variable. This variable is later accessible in the ``view.exec``
+sub-runtime and used to create an ``inet:fqdn`` node in the destination view. If more than one
+``inet:fqdn`` node with the tag ``#foo.bar`` exists, the ``view.exec`` command would be executed
+once for each node in the pipeline as expected. Again, the actual nodes would not enter the
+sub-runtime created by the ``view.exec`` command though. Conversely, the sub-query executed in the
```
Comment (therealsilence) [inline docs/synapse/userguides/storm_ref_cmd.rstorm:2493]: The sentence that starts "Conversely..." is a bit confusing to me on a few counts. The word "conversely" implies an either/or to me - so my brain is trying to read this para as "either the var $fqdn is passed to the sub-runtime to create nodes OR some var assignment in the sub-runtime gets passed back to the primary runtime/view". Also, the situation you mention - a var assigned in the sub-runtime subquery being passed to the source view - does not appear to be happening here. Maybe a second example to illustrate that process? Unless I'm missing something? (I do like this example as it's a super simple query that shows the concept nicely.)

File: docs/synapse/userguides/storm_ref_cmd.rstorm [hunk 3]
```diff
@@ -2468,6 +2468,35 @@ view.exec
 
 The ``view.exec`` command executes a Storm query in the specified view.
 
+**Behavior and Limitations**
+
+The ``view.exec`` command creates its own sub-runtime to execute a storm query in a different view.
```
[the remark this answers is not in the document]
Comment (MichaelSquires) [reply docs/synapse/userguides/storm_ref_cmd.rstorm]: I updated it to (hopefully) be a little more clear and use less "sub-runtime" references. What do you think?

File: docs/synapse/userguides/storm_ref_cmd.rstorm [hunk 4]
```diff
@@ -2468,6 +2468,35 @@ view.exec
 
 The ``view.exec`` command executes a Storm query in the specified view.
 
+**Behavior and Limitations**
+
+The ``view.exec`` command creates its own execution environment (sub-runtime) to execute a storm
```
Comment (reign-git) [inline docs/synapse/userguides/storm_ref_cmd.rstorm:2473]: Capitalize storm

Comment (MichaelSquires) [reply docs/synapse/userguides/storm_ref_cmd.rstorm:2473]: Done.

File: docs/synapse/userguides/storm_ref_cmd.rstorm (hunk 2 above)
Comment (therealsilence) [reply docs/synapse/userguides/storm_ref_cmd.rstorm]: Looks good - thanks for the clarifications!

**Review by invisig0th — APPROVED** [review]

**Pull request merged**